apps/builder/app/builder/features/blocking-alerts/alert.tsx

```diff
@@ -0,0 +1,35 @@
+import { css, Flex, Text, theme } from "@webstudio-is/design-system";
+import { AlertIcon } from "@webstudio-is/icons";
+
+const containerStyle = css({
+  position: "absolute",
+  top: 0,
+  left: 0,
+  right: 0,
+  bottom: 0,
```

Comment:
inset has good support
https://developer.mozilla.org/en-US/docs/Web/CSS/inset

Reply:
wow completely missed this property cc @taylornowotny I think this is awesome, I think this could be great for style panel as well

Reply:
It's kinda as "margin". In ui better to linked changes like in space ui.